### DISCUSSION TITLE
Syncing added audio tracks to a video

### DISCUSSION THREAD

**Discussion by lilboop** [mifi/lossless-cut#731] [Q&A]
Really simply I want to produce a short clip a from a longer video that has 3 tracks: The Video, my microphone audio, and the audio track.

I do not think losslesscut can merge audio tracks which would be helpful so my process to get a synchronized 10 second clip  from a longer video file is as follows.

1. I export all 3 tracks as separate files with losslesscut

2. I go into audacity and merge the 2 audio tracks. 

3. I clip the video at a picked keyframe, then i clip the audio at the same keyframe separately

4. I merge clipped audio with the clipped video and have successfully achieved what I wanted to do.  Is there a way to shorten this process?

Simply adding a merged audio track and applying it to the base video file and attempting to trim segments does not trim the added audio track, that's the issue im having.

**Comment by mifi** [comment]
I'm not sure what you mean by "merge" here, but if you mean mix them together, that is a lossy operation and not possible in llc.
I believe what you can do to slightly simplify is to:

1. I export all 3 tracks as separate files with losslesscut
2. I go into audacity and merge the 2 audio tracks.
3. Open the exported video track file in losslesscut
4. drag and drop the mixed audio file from audicity into losslesscut, and select add track
5. set your cutpoints and then export
